routes.py

```diff
@@ -0,0 +1,663 @@
+
+
+# ścieżka strony profilu
+@app.route('/profile/<server>/<nickname>', methods=['GET', 'POST'])
+@login_required
+def profile(server, nickname):
+    user = User.query.join(Profile).filter(Profile.nickname == nickname,
+                                           Profile.server == get_server_name_from_short(server.upper())).first()
+    serv = get_server_name_from_short(server.upper())
+    profile = Profile.query.filter_by(
+        server=serv, nickname=nickname).first_or_404()
+    return render_template('profile.html', server=server, profile=profile, user=user, get_server_short_name=get_server_short_name)
+
+
+# ścieżka drużyn stworzonych przez użytkownika,
+@app.route('/user/<username>/teams', methods=['GET', 'POST'])
+@login_required
+def yourteams(username):
+    user = User.query.filter_by(username=username).first_or_404()
+    user_profiles = Profile.query.filter(Profile.user_id == user.id)
+    user_clashteams = ClashTeam.query.join(Profile).filter(
+        Profile.id == ClashTeam.host_id, Profile.user_id == user.id).all()
+    user_profiles_data = {
+        user_profile.nickname: user_profile.server for user_profile in user_profiles}
+    guest_teams = []
+    for key, value in user_profiles_data.items():
+        guest_teams.extend(ClashTeam.query.join(Profile).filter(and_(or_(ClashTeam.toplane == key, ClashTeam.jungle == key, ClashTeam.midlane ==
+                           key, ClashTeam.bottom == key, ClashTeam.support == key), Profile.server == value, Profile.user_id != user.id)).all())
+    if len(user_clashteams) == 0:
+        user_clashteams = []
+
+    form = CreatingTeam(user_id=user.id, timerighnow=datetime.now())
+    return render_template('yourteams.html', get_server_short_name=get_server_short_name, user_clashteams=user_clashteams, guest_teams=guest_teams, form=form, user=user)
+
+
+# ścieżka pozwalająca tworzyć drużyny i przekierowywuje spowrotem na stronę drużyn użytkownika
+@app.route('/user/<username>/createteam', methods=['GET', 'POST'])
+@login_required
+def createteam(username):
+    user = User.query.filter_by(username=username).first_or_404()
+    form = CreatingTeam(user_id=user.id)
+    if form.validate_on_submit():
+        current_profile = Profile.query.filter_by(
+            nickname=form.profile.data, user_id=user.id).first()
+        new_team = ClashTeam(
+            host_id=current_profile.id,
+            clash_date=form.clash_date.data,
+            division=current_profile.division,
+        )
+        if form.role.data == 'Toplane':
+            new_team.add_top(current_profile.nickname)
+        elif form.role.data == 'Jungle':
+            new_team.add_jungle(current_profile.nickname)
+        elif form.role.data == 'Midlane':
+            new_team.add_midlane(current_profile.nickname)
+        elif form.role.data == 'Bottom':
+            new_team.add_bottom(current_profile.nickname)
+        else:
+            new_team.add_support(current_profile.nickname)
```

Comment:
Use [match case](https://www.geeksforgeeks.org/python-match-case-statement/). Also, you should have another explicit case for "Support" and show an explicit error if the provided value doesn't match any of the 5 expected positions. 

That's because one day you or someone else might make a typo on the front-end. Then, if this code errors, it's easy to find and fix. Otherwise everything will be working but with wrong positions being added - much harder to track down.